src/routes/root.jsx

```diff
@@ -0,0 +1,29 @@
+// @ts-check
+
+import { useContext } from "react";
+import { Box, Spinner } from "@primer/react";
+import { OctokitContext } from "../components/octokit-provider.js";
+import { useNavigate } from "react-router-dom";
+
+// TODO: rely on loaders and actions
+
```

Comment:
This route is only doing redirects and showing a loading spinner in the meantime but I expect this to be the homepage of Project Forms.

Reply:
In upcoming commits, it is recommended we move to `loaders` and `actions` instead, [as recommended by React Router docs](https://reactrouter.com/en/main/hooks/use-navigate):
![image](https://github.com/project-forms/project-forms.github.io/assets/2574275/a437107a-0622-4960-a5cd-caeb5eb26350)